backend/src/akvo/lumen/lib/dataset.sql

```diff
@@ -148,7 +148,8 @@ SELECT data_group.table_name AS "table-name",
    AND data_group.dataset_version_id = dataset_version_2.id
    AND dataset_version_2.version=(SELECT max(version)
                                     FROM dataset_version_2
-                                   WHERE dataset_id= :dataset-id);
+                                   WHERE dataset_id= :dataset-id)
+ORDER BY created;
```

Comment:
Do we want to add an `order` column to `data_group` instead?

Reply:
added ORDER BY data_group.created